src/crypto.rs

```diff
@@ -29,10 +29,8 @@ pub enum Algorithm {
 /// The actual HS signing + encoding
 fn sign_hmac(alg: &'static digest::Algorithm, key: &[u8], signing_input: &str) -> Result<String> {
     let signing_key = hmac::SigningKey::new(alg, key);
-    Ok(base64::encode_config(
-        hmac::sign(&signing_key, signing_input.as_bytes()).as_ref(),
-        base64::URL_SAFE_NO_PAD
-    ))
+    Ok(base64::encode_config(hmac::sign(&signing_key, signing_input.as_bytes()).as_ref(),
```

Comment:
Block style is currently recommended by the fmt-rfc folks but not the default in rustfmt. [This is the settings they recommend.](https://github.com/serde-rs/serde/blob/v1.0.0/rustfmt.toml)